packages/template-retail-react-app/app/pages/login/passwordless-landing.test.js

```diff
@@ -77,7 +60,7 @@ jest.mock('@salesforce/commerce-sdk-react', () => {
             .fn()
             .mockImplementation((helperType) => mockAuthHelperFunctions[helperType]),
         useCustomerBaskets: () => {return {data: mockMergedBasket, isSuccess: true}},
-        useCustomerType: jest.fn(() => {return {isRegistered: false, customerType: 'guest'}})
+        useCustomerType: jest.fn(() => {return {isRegistered: true, customerType: 'guest'}})
```

Comment:
the other way of doing this is by mocking the `/passwordless/token` api using `global.server.use` but because of the `/jwks` validation I was having issues. So instead I just mocked that the user is logged in